config/routes.rb

```diff
@@ -1,8 +1,18 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "pages#home"
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get "uikit", to: static("front.html.erb")
+
+  resources "users", only [:show] do
+    resources "trips", only [:new, :create, :update] do
+      resources "moments", only [:index, :create]
```

Comment:
:moments
no index but :new needed
missing the steps nested as moments
options (no need to nest):  edit, update and destroy

Reply:
i think we need index for the profile page